src/BitStringAddresses/multicomponent.jl

```diff
@@ -132,26 +132,53 @@ end
     FermiFS2C(onr_a, onr_b)
 
 Fock state address with two fermionic (spin) components. Alias for [`CompositeFS`](@ref)
-with two [`FermiFS`](@ref) components.
-Construct by specifying either two compatible [`FermiFS`](@ref)s or [`onr`](@ref)s.
+with two [`FermiFS`](@ref) components. Construct by specifying either two compatible
+[`FermiFS`](@ref)s, two [`onr`](@ref)s, or the number of modes followed by `mode =>
+occupation_number` pairs, where negative occupation numbers will go to the second
+component. See examples below.
+
+# Examples
+
 ```jldoctest
-julia> f2c = FermiFS2C((1,0,0),(0,1,1))
+julia> FermiFS2C(FermiFS((1,0,0)), FermiFS((0,1,1)))
+CompositeFS(
+  FermiFS{1,3}((1, 0, 0)),
+  FermiFS{2,3}((0, 1, 1)),
+)
+
+julia> FermiFS2C((1,0,0), (0,1,1))
+CompositeFS(
+  FermiFS{1,3}((1, 0, 0)),
+  FermiFS{2,3}((0, 1, 1)),
+)
+
+julia> FermiFS2C(3, 1 => 1, 2 => -1, 3 => -1)
 CompositeFS(
   FermiFS{1,3}((1, 0, 0)),
   FermiFS{2,3}((0, 1, 1)),
 )
 
-julia> f2c isa FermiFS2C
-true
+julia> fs"|↑↓↓⟩"
+CompositeFS(
+  FermiFS{1,3}((1, 0, 0)),
+  FermiFS{2,3}((0, 1, 1)),
+)
 ```
 """
 const FermiFS2C{N1,N2,M,N,F1,F2} =
     CompositeFS{2,N,M,Tuple{F1,F2}} where {F1<:FermiFS{N1,M},F2<:FermiFS{N2,M}}
+
 FermiFS2C(f1::FermiFS{<:Any,M}, f2::FermiFS{<:Any,M}) where {M} = CompositeFS(f1, f2)
 FermiFS2C(onr_a, onr_b) = FermiFS2C(FermiFS(onr_a), FermiFS(onr_b))
+FermiFS2C(M::Integer, pairs...) = FermiFS2C(M, pairs)
```

Comment:
```julia
julia> FermiFS2C(3, 1=> 1)
ERROR: MethodError: no method matching filter(::Rimu.BitStringAddresses.var"#46#49", ::Pair{Int64, Int64})
Closest candidates are:
  filter(::Any, ::Tuple) at tuple.jl:380
  filter(::Any, ::Array{T, N}) where {T, N} at array.jl:2534
  filter(::Any, ::AbstractDict) at abstractdict.jl:468
  ...
```
errors.

This would fix it:
```suggestion
FermiFS2C(M::Integer, pairs::Pair...) = FermiFS2C(M, pairs)
```